mne/io/tests/test_meas_info.py

```diff
@@ -141,8 +141,8 @@ def test_duplicate_name_correction():
     assert info['ch_names'] == ['A-0', 'A-1', 'A-2']
 
     # When running number is not possible
-    with pytest.raises(ValueError, match='Adding a running number'):
-        create_info(['A', 'A', 'A-0'], 1000., verbose='error')
+    info = create_info(['A', 'A', 'A-0'], 1000., verbose='error')
+    assert info['ch_names'] == ['A-a', 'A-1', 'A-0']
```

Comment:
Might be better to adjust the original failure mode test rather than replace it:
```suggestion
    with pytest.raises(ValueError, match='Adding a running alphanumeric'):
        create_info(['A'] * 27 + ['A-0'], 1000., verbose='error')
    info = create_info(['A', 'A', 'A-0'], 1000., verbose='error')
    assert info['ch_names'] == ['A-a', 'A-1', 'A-0']
```